client/stylesheets/_toCat.scss

```diff
@@ -2513,7 +2516,8 @@ i:hover {
 }
 
 .clickDropper li ul li a {
-	padding: 15px 20px 15px 15px !important;
+	padding: 15px 0px 15px 15px !important;
+    font-size: 0.8rem !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces